README.md

```diff
@@ -36,9 +42,18 @@ The `ffmpeg(.exe)` and `ffprobe(.exe)` files should be placed in `~/.stash` on m
 
 # Usage
 
+## Quickstart Guide
+1) Download and install Stash and its dependencies
+2) Run Stash. It will prompt you for some configuration options and a directory to index (you can also do this step afterward)
+3) After configuation, Stash will open your web browser and take you to the home screen.
```

Comment:
This isn't quite true. You need to navigate to the stash address, it is not done automatically.